hashicorp/consul-k8s
# Health checks base

Add base support for health checks to consul-k8s.

How I've tested this PR:
Manual testing
Unit tests and acceptance tests from consul-helm PR.

How I expect reviewers to test this PR:
run unit tests and manual test which is simply deploying a connect injected application with k8s readiness probes, then checking the UI or consul agent for health checks status.

Checklist:
